docs/user-guide/diffing.md

```diff
@@ -32,6 +32,7 @@ spec:
     - /spec/replicas
 ```
 
+Note that the `group` field relates to the kubernetes API group.
```

Comment:
```suggestion
Note that the `group` field relates to the [Kubernetes API group](https://kubernetes.io/docs/reference/using-api/#api-groups).
```

Maybe helpful?

Reply:
Plus it probably is helpful to note that it must not include the API version, only the group. I've seen a couple of times that people will specify `group` as `apps/v1` and similar.